tests/test_fact.py

```diff
@@ -0,0 +1,204 @@
+"""
+test_fact.py - Tests for Fact class
+
+test suite for Fact object class
+
+Copyright (c) 2018 The Fuel Rat Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+import pytest
+import datetime
+from datetime import timezone
+
+pytestmark = pytest.mark.fact_class
+
+
+@pytest.mark.parametrize("strings", ["Shatt", "Ion4", "290ugs03", "ThISsTRiNG is AwFuL"])
```

Comment:
Ditto the comment regarding whether getters/setters are necessary.

Reply:
Are you asking if parametrize is necessary here?  Unclear.

Reply:
Commented the wrong line, this was meant to reference all of the tests for the getters/setters, and whether or not the methods they test are necessary to begin with.

Reply:
No unicode tests?